Makefile

```
@@ -218,6 +218,8 @@ tidy::
 tidy_fix::
 	./scripts/tidy.sh
 
+renovate:: tidy_fix
```

Any reason we want multiple `renovate` target definitions?
```suggestion
renovate: tidy_fix
```